content/inferno/time-series/time-series.md

```diff
@@ -0,0 +1,309 @@
+author: clone95
+description: This guide aims to show you the Data Science applications you can build for dealing with time series data.
+---
+
+# Index
+
+- [Time Is Money](#Time-Is-Money)
+- [Introduction To Time Series](#Introduction-To-Time-Series)
+- [Time Series Problems Overview](#Time-Series-Problems-Overview)
+    - [Time Series Analysis](#Time-Series-Analysis)
+    - [Time Series Forecasting](#Time-Series-Forecasting)
+    - [Time Series Classification](#Time-Series-Classification)
+    - [Time Series Forecasting as a Classification Problem](#Time-Series-Forecasting-as-a-Classification-Problem)
+    - [The Matrix Profile](#The-Matrix-Profile)
+    - [Automatic Time Series Feature Extraction](#Automatic-Time-Series-Feature-Extraction)
+- [Conclusions](#Conclusions)
+
+
+**Disclaimer!**
+
+This guide is not intended to be exhaustive of everything you can do with time series data, but to offer you a good overview of the possible approaches you can take or things you should check when dealing with them. 
+
+Having that said, let's begin!
+
+## Time Is Money
+
+Whether it is the sale of products, the estimated production of an agricultural field, or the forecast of any activity of a company, time series are an indispensable type of data, which has been used for thousands of years to inform the activities to be done **today**, based on what will happen **tomorrow**. 
+
+A good estimate of the future, for a company, can bring a great saving of time (and money)! 
+
+If you ask any manager will tell you that _time is even more important than money_, because you can convert time into money, _but the opposite is not possible!_ 
+
+Learning to work with time series is a very valuable skill in the real world, and applicable to many situations and problems. 
+
+It is also considered one of the most "difficult" types of data to manage since _it has additional complexities_ compared to more traditional data such as tabular data or text.
+
+We will see these additional complexities in the section [Time Series Additional Challenges](#Time-Series-Addtional-Challenges).
+
+Now that you know why it's important to know how to work with time series, it's time to start seeing them in practice! 
+
+## Introduction To Time Series
+#### Know the Time Series Data Structure
+
+The first resources Virgilio recommends to get you started are the following:
+
+- [Tutorial: Time Series Analysis with Pandas](https://www.dataquest.io/blog/tutorial-time-series-analysis-with-pandas/)
+- [Working with Time Series](https://jakevdp.github.io/PythonDataScienceHandbook/03.11-working-with-time-series.html)
+
+These are an exhaustive introduction to the type of "time" data, how it is manipulated with Pandas (indexing, transformation, visualization).
+
+After you've gone through them, you should understand how to write **"idiomatic Pandas"** for time series, and you can find a tutorial (section 7 of [Modern Pandas](https://tomaugspurger.github.io/modern-1-intro)) here: 
```

Comment:
And also say "you will have a better understanding of what time-series mean and what time-series data is about"?